rtf_vis_tool/src/Components/LandingPageGraph.js

```diff
@@ -76,15 +75,15 @@ const DivGraph = (props) => {
     const toggleRotSummaryDisplay = (bool) => {setShowRSS(bool)};
 
     return (
-        <div className="div_graph_container">
+        <div className="lp_graph_container">
             <div className="navbar">
                 <h2 className="gtsfm_header">GTSFM Computational Graph Visualizer</h2>
             </div>
 
             {/* Render popups only when the respective node is clicked*/}
             {/* {showFS && <FrontendSummary json={fs_json} toggleFS={toggleFrontEndSummaryDisplay}/>} */}
             {/* {showMVO && <MVOSummary json={mvo_json} toggleMVO={toggleMVOMetrics}/>} */}
-            {showDA_PC && <Data_Association_PC da_json={da_json} rotated_json={rotated_da_json} toggleDA_PC={toggleDataAssoc_PointCloud}/>}
+            {showDA_PC && <Bundle_Adj_PC da_json={da_json} rotated_json={rotated_da_json} toggleDA_PC={toggleDataAssoc_PointCloud}/>}
```

Comment:
is the code above meant to be commented out?
```
            {/* Render popups only when the respective node is clicked*/}
            {/* {showFS && <FrontendSummary json={fs_json} toggleFS={toggleFrontEndSummaryDisplay}/>} */}
            {/* {showMVO && <MVOSummary json={mvo_json} toggleMVO={toggleMVOMetrics}/>} */}
```

Reply:
yes. I have just deleted a few of those commented out components entirely for the PR. I'll add them back in the next one.